ExploreASL_Master.m

```diff
@@ -1,37 +1,54 @@
-function [x] = ExploreASL_Master(DataParPath, ProcessData, SkipPause, iWorker, nWorkers, iModules)
-%ExploreASL_Master ExploreASL pipeline master wrapper calling the individual pipeline modules
+function [x] = ExploreASL_Master(varargin)
+%ExploreASL_Master ExploreASL pipeline master wrapper calling the individual import & pipeline modules
 %
-% FORMAT: [x] = ExploreASL([DataParPath, ProcessData, SkipPause, iWorker, nWorkers, iModules])
+% FORMAT: [x] = ExploreASL([DataParPath, ImportArray, ProcessArray, SkipPause, iWorker, nWorkers])

```

Comment:
Good that you keep the parameter names at line four - makes it easier to see what's there.

Reply:
Similar question to the one about the new field for x.StudyRoot. I used x.SkipPause, x.iWorker and x.nWorkers to make the input parsing easier. If that's a problem or you want them in sub-fields of x we can easily do this. Separate variables are just a bit problematic and make the code really ugly there.